### PR TITLE
Add build specific files/updates for Hera and Gaea C6

### PR DESCRIPTION
**Pull request by AnnaSmoot-NOAA**
## DESCRIPTION OF CHANGES: 
In order for the latest version of AQM-Workflow (updated with 9km grid) to build on Hera and Gaea C6 the following files need to be changed or added: 

- app_build.sh (updated file: works for Hera, C6, and WCOSS)

- build_hera_intel.lua (additional file)

- build_gaeac6_intel.lua (additional file)

- build_gaeac6.ver (additional file)

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Scientific results will change based on this work 

## TESTS CONDUCTED: 

- [X] gaea.intel
- [X] hera.intel
- [ ] wcoss2.intel

## DEPENDENCIES:
N/A

## DOCUMENTATION:
N/A

## ISSUE: 
Addresses issues: #41 #42 #6 

## CHECKLIST

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [X] My changes do not require updates to the documentation (explain).
- [X] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] Any dependent changes have been merged and published

## PRIOTIRY LEVEL

- [ ] Critical: emergency fix for a major issue
- [ ] High: upcoming production fix/change 
- [X] Normal: important but not high priority
- [ ] Low: routine addition or bonus feature that can wait

### CODE REVIEW

File: sorc/app_build.sh [hunk 1]
```diff
@@ -113,6 +116,7 @@ ENABLE_OPTIONS=""
 DISABLE_OPTIONS=""
 BUILD_TYPE="Release"
 BUILD_JOBS=4
+EXTRN=false
```
Comment (BrianCurtis-NOAA) [inline sorc/app_build.sh]: i removed the EXTRN parts of this script because we don't need them anymore.

File: sorc/app_build.sh [hunk 2]
```diff
@@ -155,6 +159,8 @@ while :; do
     --enable-options|--enable-options=) usage_error "$1 requires argument." ;;
     --disable-options=?*) DISABLE_OPTIONS=${1#*=} ;;
     --disable-options|--disable-options=) usage_error "$1 requires argument." ;;
+    --extrn) EXTRN=true ;;
+    --extrn=?*|--extrn=) usage_error "$1 argument ignored." ;;
```
Comment (BrianCurtis-NOAA) [inline sorc/app_build.sh]: Here as well

File: sorc/app_build.sh [hunk 3]
```diff
@@ -223,6 +229,37 @@ fi
...
+  fi
+  if [ -d "${SORC_DIR}/UPP" ]; then
+    printf "... removing UPP ...\n"
+    rm -rf "${SORC_DIR}/UPP"
+  fi
+
+  # run check-out
+  printf "... checking out external components ...\n"
+  ./manage_externals/checkout_externals
+fi
```
Comment (BrianCurtis-NOAA) [inline sorc/app_build.sh]: Here as well

File: sorc/app_build.sh [hunk 4]
```diff
@@ -76,6 +78,7 @@ Settings:
   CCPP=${CCPP_SUITES}
   ENABLE_OPTIONS=${ENABLE_OPTIONS}
   DISABLE_OPTIONS=${DISABLE_OPTIONS}
+  EXTRN=${EXTRN}
```
Comment (BrianCurtis-NOAA) [inline sorc/app_build.sh]: Here as well

File: sorc/app_build.sh [hunk 5]
```diff
@@ -25,6 +25,8 @@ OPTIONS
   --disable-options="OPTION1,OPTION2,..."
       disable ufs-weather-model options; delimited with ','
       (e.g. 32BIT | INLINE_POST | UFS_GOCART | MOM6 | CICE6 | WW3 | CMEPS)
+  --extrn
+      check out external components
```
Comment (BrianCurtis-NOAA) [inline sorc/app_build.sh]: Here as well.

**Review by BrianCurtis-NOAA — CHANGES_REQUESTED** [review]
Please remove the EXTRN sections, as we don't need them any more.

File: sorc/app_build.sh (hunk 5 above)
Comment (AnnaSmoot-NOAA) [reply sorc/app_build.sh]: I've made all these changes to app_build.sh on Hera and successfully pushed everything back to AnnaSmoot-NOAA:build_updates! How do I update the code in this PR?

**Comment by BrianCurtis-NOAA** [comment]
If you push to AnnaSmoot-NOAA build_updates branch this PR will automatically update with those changes. I don't see those changes, yet.

**Comment by AnnaSmoot-NOAA** [comment]
I swore that push went through yesterday, but I see that it didn't. It looks like it went through this time.

**Review by BrianCurtis-NOAA — APPROVED** [review]

**Pull request merged**